docs/release-notes.md

```diff
@@ -10,15 +10,22 @@ hide:
 ### Added
 
 - allow multiple ForeignKeys with the same related_name in one model
+- support ForeignKeys on multiple columns
```

Comment:
Please make sure everything in the release notes is capitalised? There are plenty of examples there too.